content/blog/posts/A-Message-from-the-Director.rst

```diff
@@ -0,0 +1,66 @@
+A Message from the Director
+===========================
+:date: 2017-08-15
+:slug: a-message-from-the-director
+:author: Lance Albertson
+:img: OSLSignPicture.jpg
+:tag: featured-stories
+
+Since its creation in 2002, the Open Source Lab (OSL) has been a beacon of
+innovation, experiential learning, and hosting at Oregon State University and
+beyond. As director, I’ve had the pleasure of watching the lab and its employees
+grow over the years and I’m so grateful to have been given such an opportunity.
+I want to express why the OSL is important not only to the open source
+community, but to our larger academic institution and its students.
+
+Becoming director of the OSL has been one of the most rewarding positions I have
+ever had the pleasure of filling. I have had the opportunity to watch students
+with amazing potential grow into graduates with incredible opportunities, all
+because of their hard work. As much as I would like to say the OSL is
+responsible for this transformation, I must give credit where credit is due:
+The students who come through the OSL are passionate, driven, and highly
+intelligent. They’re intelligent, patient, and generous in time and knowledge.
+Time and time again, I have seen the entire lab band together to help solve some
+of our most perplexing problems, from the most experienced mentors to the fresh
+eyes of our newest hires discussing openly and enthusiastically.
+
+.. image:: /images/OSLWorkers3.jpg
+    :scale: 100%
+    :align: right
+    :alt: OSL Student Employees
+
+We have faced many challenges and changes but the students’ commitment to open
+source, the lab, and each other has never faltered. In addition, I want to take
+a moment to recognize the committed full-time staff here at the lab, both past
+and present. The success and growth of our students is directly related to the
+dedication of our mentors. Though they bring varying perspectives and skills to
+the lab, they all firmly believe in the ideology of our community and this
+passion transcends through the lab from mentor to student and beyond. The
+prestige and prosperity of the lab would not be possible without their help.
+
+Over the years, I have had the pleasure of seeing many of our students grow to
+be innovative leaders of the open source community’s next generation. For
+example, Alex Polvi and Brandon Philips co-founded CoreOS, an organization
+dedicated to open access and internet security. Emily Dunham was nominated for
+the Women in Open Source Award for 2015 and works at Mozilla on supporting the
+infrastructure behind the Rust language. Sarah Cooley works for Microsoft as a
+Program Manager for SQL and Hyper-V. And these are just to name a few. Here at
+the OSL, we are very proud of all our former students’ accomplishments and we
+hope to continue to provide our current and future students with all the tools
+and skills they need to exceed their goals.
```

Comment:
I'd replace everything after "And these are just ..." with:

"These are just a few of many former OSL students’ accomplishments. The OSL hopes to continue to provide our current and future students with the tools and skills they need to exceed their goals."